components/ElectronicsDropOffCard.js

```diff
@@ -1,27 +1,30 @@
 import React from 'react';
 
-import Card, { CardAddress, CardParagraph, CardTitle } from './Card';
+import Card, { CardAddress, CardParagraph } from './Card';
 
 function ElectronicsDropOffCard() {
   return (
     <Card
+      address={
+        <CardAddress label="Email us to learn more at:">
+          {
+            <a href="mailto:hi@mhfcares.org?subject=Electronics%20Drop-Off%20Box">
+              {'hi@mhfcares.org'}
+            </a>
+          }
+        </CardAddress>
+      }
       backgroundImageAltText="Help families get access to technology"
       backgroundImageSource="/images/ElectronicsDrop.jpg"
-    >
-      <CardTitle>{'Electronics Drop-Off Box'}</CardTitle>
-      <CardParagraph>
-        {
-          "Any organization or business around the world can help families get access to technology. Put a donation box in your office for used laptops, tablets, and smartphones. Volunteers will pick up the items periodically, fix them up, and provide them to families in need. You can optionally be involved with refurbishment and placement too if you'd like."
-        }
-      </CardParagraph>
-      <CardAddress label="Email us to learn more at:">
-        {
-          <a href="mailto:hi@mhfcares.org?subject=Electronics%20Drop-Off%20Box">
-            {'hi@mhfcares.org'}
-          </a>
-        }
-      </CardAddress>
-    </Card>
+      paragraph={
+        <CardParagraph>
```

Comment:
This component seems to only wrap its children in a span tag. Maybe we can refactor here and anywhere else this is being used.

Reply:
Yeah, so these components whose names are ending with the card, I was trying to make them generic and then I put all their data in JSON file. So, my final plan is to delete them as we have the data.